.github/workflows/python-integration.yml

```diff
@@ -53,17 +54,13 @@ jobs:
           aws-region: ${{ vars.S3_REGION }}
 
       - name: Set up stable Rust
-        uses: actions-rs/toolchain@v1
-        with:
-          toolchain: stable
-          override: true
+        uses: dtolnay/rust-toolchain@stable
```

Comment:
Why have we changed this? 
Should we update the other workflows too: https://github.com/search?q=repo%3Aawslabs%2Fs3-connector-for-pytorch%20actions-rs%2Ftoolchain%40v1&type=code?